packages/design-system-scripts/cli.js

```diff
@@ -20,24 +26,60 @@ yargs
     },
     handler: async argv => {
       const { build } = require('./gulp/build');
-      await build(argv.sourcePackageDir);
+      await build(argv.sourcePackageDir, { ...argv });
+    }
+  })
+  .command({
+    command: 'build-docs <sourcePackageDir> <docsPackageDirs..>',
+    desc: 'Builds your main design-system package and its corresponding documentation site',
+    builder: yargs => {
+      describeSourcePackageDir(yargs);
+      describeDocsPackageDirs(yargs);
+
+      yargs
+        .option('rootPath', {
+          default: '',
+          description:
+            'The path of the docs site relative to the domain root. For example, if your docs site is hosted at www.domain.com/design/ your rootPath would be `design/`'
+        })
+        .option('githubUrl', {
+          default: '',
+          description:
+            'The base path for your GitHub repository URLs. This is used to render links to releases, issues, etc.'
+        });
```

Comment:
I defined these for this command only because I figured neither of them would need to be customized for the dev server. Root path would always be `''` for hosting at the root of `localhost:3000`, and the githubUrl really doesn't matter.

Reply:
Ah, were these customizable in the original scripts? and when would we define `githubUrl`?

Reply:
This was not customizable in the original script: https://github.com/CMSgov/design-system/blob/master/packages/docs/src/scripts/shared/githubUrl.js#L2